Resources/Private/Fusion/Root.fusion

```diff
@@ -1,4 +1,7 @@
 ##
 # Include all .fusion files
 #
-include: ./**/*.fusion
+include: **/*.fusion
+// TODO Remove this line when the following PR is merged:
+// https://github.com/neos/neos-development-collection/pull/3903
+include: ../../../NodeTypes/**/*.fusion
```

Comment:
The pull request was merged. We should change this line.
It should work with `include: nodetypes://Neos.Demo/**/*.fusion`